src/Renderer/BasicMaterial.js

```diff
@@ -51,7 +51,7 @@ function BasicMaterial(color) {
         uuid: { value: 0 },
         debug: { value: false },
         selected: { value: false },
-        lightOn: { value: true },
+        enableLighting: { value: true },
```

Comment:
This uniform is a status, not an action. So `lightingOn` or `lightingEnabled` would be a better name.